commands/FBDebugCommands.py

```diff
@@ -189,6 +191,83 @@ def run(self, arguments, options):
     fb.evaluateEffect('[[UIApplication sharedApplication] performSelector:@selector(_performMemoryWarning)]')
 
 
+def switchBreakpointState(expression,on):
+
+  expression_pattern = re.compile(r'{}'.format(expression),re.I)
+
+  target = lldb.debugger.GetSelectedTarget()
+  for breakpoint in target.breakpoint_iter():
+    if breakpoint.IsEnabled() != on and (expression_pattern.search(str(breakpoint))):
+      print str(breakpoint)
+      breakpoint.SetEnabled(on)      
```

Comment:
Did you find this was necessary in your experience? I'm curious what are the cases where these commands need to apply to the breakpoint itself instead of working only on the locations.

Reply:
If we use `br disable 41` to off `41`,and `benable 41` only enable locations's state.
The `41` `41.1` will be ignore.